src/locale/locales/de/messages.po

```diff
@@ -1595,7 +1595,7 @@ msgstr "Passwort vergessen"
 #: src/screens/Hashtag.tsx:108
 #: src/screens/Hashtag.tsx:148
 msgid "From @{sanitizedAuthor}"
-msgstr ""
+msgstr "Aus @{sanitizedAuthor}"
```

Comment:
```suggestion
msgstr "Von @{sanitizedAuthor}"
```

`aus` is referring to a place, `von` is used as 'from' for people

Reply:
Ah I see, that makes sense